src/controllers/users.controller.js

```diff
@@ -0,0 +1,50 @@
+import dotenv from 'dotenv';
```

Comment:
Please rename this file to UserController

Reply:
Yeah I did it